packages/components-js/tests/e2e/specs/select.e2e.ts

```diff
@@ -1633,3 +1356,116 @@ test.describe('optgroups', () => {
     }
   });
 });
+
+test.describe('form', () => {
+  test('should include name & value in FormData submit', async ({ page }) => {
```

Comment:
We should also test if this works when the value is changed dynamically either by keyboard, mouse or programmatically. Same for `multi-select`